src/commands/logs.rs

```diff
@@ -0,0 +1,208 @@
+use std::ops::Sub;
+use std::time::Duration;
+
+use anyhow::{bail, Context, Result};
+use chrono::Utc;
+use cloud::{CloudClientExt, CloudClientInterface};
+use cloud_openapi::models::Entry;
+use std::option::Option;
+
+use crate::commands::create_cloud_client;
+use crate::commands::deploy::SPIN_DEPLOY_CHANNEL_NAME;
+use crate::opts::*;
+use clap::Parser;
+use uuid::Uuid;
+
+/// fetch logs for an app from Fermyon Cloud
+#[derive(Parser, Debug)]
+pub struct LogsCommand {
+    /// Find app to fetch logs for in the Fermyon instance saved under the specified name.
```

Comment:
@rajatjindal forgive me for nagging on this but I would strongly encourage changing this to something generic like 

```suggestion
    /// Use the Fermyon instance saved under the specified name.
```

The mention of "app" makes it sound like the app, a la "find the app blah blah the specified name"

Reply:
thank you for being persistent. I had intended to address this feedback, but it seems like it got lost in other comments while i was going through the comments.